tsconfig.json

```diff
@@ -9,5 +9,10 @@
     "removeComments": false,
     "noImplicitAny": true,
     "suppressImplicitAnyIndexErrors": true
-  }
+  },
+  "exclude": [
```

Comment:
Yes, but not really works. The AngularLanguageService read the whole node_modules :( I try to find the good solution. I think the best if we convert the shared folder into angular module.
more than 1 min while the IDEs (intellij, vscode) load the whole project.

@mondalaci @fjozsef @attilacsanyi What do you think about it?